metabase/metabase
# When updating an alert, don't also update the question

Fixes #36395

### Description

On master, when you edit an alert, an additional `PUT` request is to send to update the card. This is not needed, because you can't edit an alert without first saving the question. On this branch, this request is not done.

### How to verify

On master:

1. Set up a new card and a new alert
2. Edit the alert
3. Note in the DevTools Network tab the PUT request to /api/card/:id

On this branch, do the same thing and note that there is no additional request.

### Demos

* [On master, updating the alert triggers a `PUT` request to /api/card:id](https://www.loom.com/share/4090bee88bf242abad12ff5b8b94b4f4?sid=98ef6bec-0d7a-48be-a155-b3f7ec794c62)
* [On this branch, updating the alert does not trigger a `PUT` request to /api/card/:id](https://www.loom.com/share/0558f085f70e40508e24642c152e3094?sid=a48ffa8a-5698-4765-81fc-5f311e8aeb98)

### Checklist


<s>Since this PR removes unneeded functionality, I haven't written a test for this. Of course, reviewers, please let me know if you think a test would make sense here.</s> I've now written a test, per @npfitz's request.
